### PR TITLE
Propagate photon_series_type to BaseImagingExtractorInterface 

### PR DESCRIPTION
**Pull request by alessandratrapani**
Propagate photon_series_type to BaseImagingExtractorInterface get_metadata() and get_metadata_scehma()

### CODE REVIEW

**Comment by CodyCBakerPhD** [comment]
@alessandratrapani I was thinking this could be in the base for all imaging interfaces -what do you think?

**Comment by alessandratrapani** [comment]
> @alessandratrapani I was thinking this could be in the base for all imaging interfaces -what do you think?

Do you mean like in the last commit?

**Comment by CodyCBakerPhD** [comment]
> Do you mean like in the last commit?

Yep, just need to fix (and add some new) tests

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,7 @@
 * Fixed a bug with `parameterized` and `pytest-xdist==3.6.1` in the `ScanImageImagingInterface` tests: [PR #829](https://github.com/catalystneuro/neuroconv/pull/829)
 * Added `XX` and `XO` to the base metadata schema. [PR #833](https://github.com/catalystneuro/neuroconv/pull/833)
 * `BaseImagingExtractor.add_to_nwbfile()` is fixed in the case where metadata is not supplied [PR #849](https://github.com/catalystneuro/neuroconv/pull/849)
+* Propagated `photon_series_type` to `BaseImagingExtractorInterface` init instead of passing it as an argument of `get_metadata()` and `get_metadata_schema()`
 
 ### Improvements
 * Fixed docstrings related to backend configurations for various methods. [PR #822](https://github.com/catalystneuro/neuroconv/pull/822)
```
[the remark this answers is not in the document]
Comment (alessandratrapani) [reply CHANGELOG.md:33]: yes right sorry!

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**